test/e2e/specs/adding-blocks.test.js

```diff
@@ -33,14 +33,27 @@ describe( 'adding blocks', () => {
 		return page.mouse.click( x, y );
 	}
 
+	async function clickBelow( elementHandle ) {
+		await elementHandle._scrollIntoViewIfNeeded();
+		const box = await elementHandle._assertBoundingBox();
```

Comment:
These two functions seem private (`_`) Why don't you use https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#elementhandleboundingbox instead?

Reply:
Good call. This was copied and adapted from the above temporary `clickAtRightish`, which itself was adapted from Puppeteer's internal `ElementHandle#click`, hence the internal functions. Since I don't think this function is temporary, updated to use the public API instead in the rebased 46178e9.